cloud-azure/src/main/resources/templates/arm-network-dnszone.ftl

```diff
@@ -0,0 +1,61 @@
+<#setting number_format="computer">
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
```

Comment:
this seems pretty old, can't we move to a newer one?

Reply:
I update it for this template and file a follow-up to update all the others, as every arm template references this schema version except for 1.